go/vt/vtgate/planbuilder/testdata/tpcc_cases.txt

```diff
@@ -858,7 +858,7 @@ Gen4 plan same as above
     },
     "TargetTabletType": "PRIMARY",
     "MultiShardAutocommit": false,
-    "Query": "update order_line1 set ol_delivery_d = NOW() where ol_o_id = 235 and ol_d_id = 315 and ol_w_id = 8",
+    "Query": "update order_line1 set ol_delivery_d = now() where ol_o_id = 235 and ol_d_id = 315 and ol_w_id = 8",
```

Comment:
This should have happened on merging main

Reply:
Test TPCC in our plan_test is failing on main, because of 2 merges, one PR made the change to add parsing for `NOW`, which made the parser output be `now` , and another PR which added an additional test to TPCC cases having `NOW` in the query, the result in the file expects `NOW` but we are getting `now`. So this change is alright.